cirq/google/calibration/phased_fsim.py

```diff
@@ -304,6 +304,16 @@ class FloquetPhasedFSimCalibrationOptions(PhasedFSimCalibrationOptions):
 )
 
 
+"""PhasedFSimCalibrationOptions with zeta and gamma angles characterization requests set to True."""
+ZETA_GAMMA_FLOQUET_PHASED_FSIM_CHARACTERIZATION = FloquetPhasedFSimCalibrationOptions(
```

Comment:
add documentation (or even reflect in the name of this object!) that these options are for situations that can be remedied with single-qubit Z rotations.

Reply:
Done